app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.startswith("cp"):
+        split_command = command.split()
```

Comment:
as for me, it's better to unpack values into descriptive variables, for example command, old_file, copy_file = command.split()